using-mobile-extensions/adobe-campaign-standard/README.md

```diff
@@ -184,6 +184,13 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 Need help creating an in-app message using Adobe Campaign? For more information, see [Preparing and sending an In-App message](https://helpx.adobe.com/campaign/standard/channels/using/preparing-and-sending-an-in-app-message.html).
 {% endhint %}
 
+{% hint style="warning" %}
+If developing an Android application, the Campaign Standard Extension's FullscreenMessageActivity must be added to your AndroidManifest.xml file to properly display fullscreen in-app messages. Add this line to the AndroidManifest.xml file:
```

Comment:
Please reword so that it reads as follows:

"If you are developing an Android application, to correctly display fullscreen in-app messages, add the Campaign Standard extension's `FullscreenMessageActivity` to your AndroidManifest.xml file:"